test/flow.tcl

```diff
@@ -83,6 +83,8 @@ set_routing_layers -signal $global_routing_layers \
   -clock $global_routing_clock_layers
 set_macro_extension 2
 
+write_db /tmp/gcd_sky130hd.db
```

Comment:
Do we intend to leave this in?

Reply:
For me, this line is super important. But I didn't go over the whole process and don't know if it is necessary for the whole project. I will delete it. But I will take care of this line and add it again if I find it useful later.